bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/EnergyApi.java

```diff
@@ -26,15 +31,46 @@
 
 @NonNullByDefault
 public class EnergyApi extends RestManager {
+    public static final String URL_HOMESTATUS = "homestatus";
+    public static final String URL_HOMESDATA = "homesdata";
```

Comment:
Actually I think it should be the other way round and all API-calls to the the "Engery"-API should be in the EnergyApi.java. Especially these two essential calls only exist in the EnergyAPI.
Eventually the "HomeApi" should vanish from the binding as it does not exist anymore with the new APIs

Reply:
I think HomeApi still exists for Security purpose. I would prefer having all calls done with a homeId paremeter landing in HomeAPI

Reply:
Not sure if that is the right way forward. All calls in Energy reference the home with just the exception of historic data from the Thermostat (/getmeasure call) - if one got a Thermostat.
So it would end up in just a "HomeApi" in the naming of the binding and no EnergyAPI (or just processing the one call ripped out of context (compared to Netatmos dev docu) with no reference at all to the "Security" and "Energy" APIs on Netatmo's dev site.

Reply:
Yes, you may be right, maybe HomeApi has to be dispatched between Security and Energy.

Reply:
So, thinking more about it: the "homesdata" lives in EnergyAPI per Netamo docs, but it is in fact the only way to retrieve modules of a house. 
What I do not understand in the binding: default scope (empty as in HomeApi) is just "read_station" which allows to only access & discover weatherstation modules. Where do the rights come from to retrieve home&security modules while doing the discovery ?
Nevertheless: I think retrieving home topology and modules should be only done in HomeAPI (as it is today), to have the single point used for discovery. So I'll remove that in EnergyAPI so that its use is consistent.
All other calls from Netatmos Energy-API description I'll put in EnergyAPI.java. 
WDYT ?

Reply:
I thing this is good and consistent.